xbmc/filesystem/NFSFile.cpp

```diff
@@ -586,21 +585,37 @@ bool CNFSFile::Open(const CURL& url)
 
   std::unique_lock<CCriticalSection> lock(gNfsConnection);
 
-  if(!gNfsConnection.Connect(url, filename))
-    return false;
+  auto NfsOpen = [this](const CURL& url, std::string& filename) -> int
```

Comment:
You seem to return bools in different locations (implicit conversion to int) and don't seem to use the return code of nfs_open. Why not make the lambda return bool right away?

Reply:
yea, I originally was going to check the ret value, but it seems it's not needed at all. I'll update it to `bool`